src/find_email_addresses.py

```diff
@@ -0,0 +1,63 @@
+import sys
+import argparse
+import validators 
+import requests
+import requests.exceptions
+from bs4 import BeautifulSoup 		# wow, not working with Python3.6, using Python3.5 
+import re
+from collections import deque
+email_pattern = re.compile("[\w\.-]+@[\w\.-]+")
```

Comment:
Nice job including other common characters beyond alphanumeric

Reply:
Credit goes to  http://regexlib.com/DisplayPatterns.aspx